config/base/operator-controller/crd/bases/olm.operatorframework.io_clusterextensions.yaml

```diff
@@ -56,6 +56,14 @@ spec:
             description: spec is an optional field that defines the desired state
               of the ClusterExtension.
             properties:
+              config:
+                description: config can be used to pass parameters to the underlying
+                  runtime.
+                items:
+                  type: object
+                  x-kubernetes-embedded-resource: true
+                  x-kubernetes-preserve-unknown-fields: true
```

Comment:
In Kubernetes v1.22+, it was removed entirely right?
I think we need to see how we should to do that now

Reply:
@thetechnick I think we might need instead

   // +kubebuilder:pruning:PreserveUnknownFields
    // +kubebuilder:validation:Schemaless
    Config map[string]interface{} json:"config,omitempty"